calendar.php

```diff
@@ -2548,19 +2548,21 @@
 			// Is the current day
 			if($day.$calendar_month.$year == $today && $month == $calendar_month)
 			{
-				$day_class = "trow_sep";
+				$current_day = "trow_sep";
```

Comment:
With these separate templates, I wonder if we can get rid of these hard-coded CSS classes in the PHP code and just put the classes in the templates?

Reply:
I can take a look but I want to get a fix for #3753 first since that's an important one 👍